templates/repo/editor/patch.tmpl

```diff
@@ -14,8 +14,8 @@
 					<div class="breadcrumb-divider">:</div>
 					<a class="section" href="{{$.BranchLink}}">{{.BranchName}}</a>
 					<span>{{ctx.Locale.Tr "repo.editor.or"}} <a href="{{$.BranchLink}}">{{ctx.Locale.Tr "repo.editor.cancel_lower"}}</a></span>
-					<input type="hidden" id="tree_path" name="tree_path" value="" required>
-					<input id="file-name" maxlength="500" type="hidden" value="diff.patch">
+					<input type="hidden" name="tree_path" value="__dummy_for_EditRepoFileForm.TreePath(Required)__">
+					<input id="file-name" type="hidden" value="diff.patch">
```

Comment:
Accidentally remove `maxlength="500"` after #29389 merged?

Reply:
It is removed intentionally.

It doesn't make sense to use maxlength for a hidden input.